### DISCUSSION TITLE
Add Block postion

### DISCUSSION THREAD

**Discussion by wlshaffer** [postmanlabs/postman-flows#54] [Defect]
When I click to `+ Add Block` in the UI the modal appears in the top right corner and I can not move it. 
![image](https://user-images.githubusercontent.com/11655615/138312847-db77881c-17be-493a-8e9a-65eed27e4c34.png)

**Comment by anudeepmedicharla** [comment]
Hey @wlshaffer, Thanks for reporting this bug. We'll fix this week :) 
cc: @dev-kanishk

**Comment by dev-kanishk** [comment]
Hey @wlshaffer, Can you please tell us on which platform you are facing this issue? Desktop App or the Web. If Desktop can you share your App version and for Web can you share the browser you are using.

**Comment by relkovn** [comment]
Hey @dev-kanishk I had the same issue when using the Desktop version.  I was using Postman v9.1.3.  I tried using the Web Version in Edge and Chrome, they both displayed the list of available blocks by the Add Block button (as it should, from the looks of it), while in the Desktop version it displays in the top left of the display.
Now, @wlshaffer  did mention the concern that he could not move it from that location, but I think that this is intended but needs to display by the button.
